blocks/log/json.py

```diff
@@ -0,0 +1,54 @@
+from mimir import Logger
+
+from .log import TrainingLogBase
+
+
+class PiclableLogger(object):
+    def __init__(self, filename, **kwargs):
+        logger = Logger(**kwargs)
+        self.__dict__ = logger.__dict__
```

Comment:
That seems like it will alias the dict object itself, is that what you want? I'm not sure that it won't have weird consequences. You could `self.__dict__.update(logger.__dict__)` otherwise.